content/shell/android/shell_apk/src/org/chromium/content_shell_apk/ContentShellActivity.java

```diff
@@ -73,10 +73,6 @@ protected void onCreate(final Bundle savedInstanceState) {
         mWindowAndroid.setAnimationPlaceholderView(
                 mShellManager.getContentViewRenderView().getSurfaceView());
 
-        // TODO(cobalt, b/376148547): set Chrobalt initial url and remove this function.
```

Comment:
nit: this should be `mStartupUrl = getUrlFromIntent(getIntent());` in this line.

Reply:
Thanks!